rider-fsharp/build.gradle.kts

```diff
@@ -338,6 +338,7 @@ tasks {
 
     withType<Test> {
         useTestNG()
+        jvmArgs = listOf("-Didea.force.use.core.classloader=true")
```

Comment:
Since this is a temporary workaround and we know it's fixed in upcoming `gradle-intellij` update, it'd be wise to add a comment to remove it later.